app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -308,6 +331,24 @@ private void updateViews() {
         }
     }
 
+    private void updateOverviewVideo() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+            if (mVideoBanner != null) {
+                mVideoBanner.setEnableAutomaticInitialization(false);
+                getLifecycle().addObserver(mVideoBanner);
+                mVideoBanner.addYouTubePlayerListener(new AbstractYouTubePlayerListener() {
+                    @Override
+                    public void onReady(@NotNull final YouTubePlayer youTubePlayer) {
+                        if (mTool != null && mTool.getOverviewVideo() != null) {
```

Comment:
Is this guaranteed based on how this is being called? Or perhaps its not because its Android and these are class attributes?

Reply:
@wrandall22 I can't guarantee when `onReady` is called.  So to prevent null pointer errors I am checking the values before use.